kratos/processes/calculate_embedded_nodal_variable_from_skin_process.h

```diff
@@ -723,27 +735,38 @@ class CalculateEmbeddedNodalVariableFromSkinProcess : public Process
 
     void AddEdgeNodes(
         const Geometry<Node<3>> &rEdgeGeometry,
-        ModelPart &rModelPart) const
+        ModelPart &rModelPart,
+        std::unordered_map<unsigned int, Node<3>::Pointer>& rNodesMap
+        ) const
     {
+        const auto& pvar_list = rModelPart.pGetNodalSolutionStepVariablesList();
```

Comment:
```suggestion
        const auto& rp_var_list = rModelPart.pGetNodalSolutionStepVariablesList();
```
(minor) Naming convention in style guide.